test/v3/amendPeriodSummary/AmendPeriodSummaryServiceSpec.scala

```diff
@@ -87,53 +70,6 @@ class AmendPeriodSummaryServiceSpec extends ServiceSpec {
 
         await(service.amendPeriodSummary(requestDataWithCl290Enabled)) shouldBe Right(ResponseWrapper(correlationId, ()))
       }
-
-      "a valid request is supplied with cl290 feature switch disabled" in new Cl290Disabled {
```

Comment:
These tests are valid and should be kept this CL290 both tests will work with the changes to the disabled private vals in the comment above